docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -79,10 +79,111 @@ on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4Q
+categories are Hispanic or Latino and Not Hispanic or Latino. 
+
+By having a set standard, 
+it improves the chance that multiple data sources will have the same 
+categories. However, this is not guarunteed. Here, we will go through 
+a few scenarios of how race and ethnicity categories might be misaligned 
+between datasets and the options for handling each. 
+
+**Nested Categories**
+
+In this guide, we use nested categories to mean that one dataset has more granular 
+categories than another. For example, one dataset might report a race group 
+as simply "Asian" whereas another might have many categories that fit within 
+this group like Chinese, Filipino, Japanese, or Korean. In fact, the US Census 
+often collects both the more granular and rolled up categories of data. 
+
+Granular data is usually better, as different groups within a single race 
+category can have very different experiences and needs. Try to keep the 
+most granular data possible. However, granular data can sometimes create 
+issues with statisitcal power or small sample size. Weigh the different 
+pros and cons as they relate to your project, centering people's 
+identities and needs in the conversation. 
+
+**Non-Nested Categories**
+
+If you need to combine multiple datasets, you will likely have to combine 
+race and ethnicity categories that won't match up perfectly. One option 
+is to take the "least common denominator" approach, which essentially means 
+use the most granular race and ethnicity categories possible that still 
+captures the data fully. 
+
+Often this will end up being the US standard categories above. We strongly 
+recommend against using fewer categories than the US standard unless it 
+is impossible to do otherwise. Creating larger buckets ends up merging 
+individuals with diverse experiences, backgrounds, and identities. Consider 
+what there is to be gained from this analysis and if race and ethnicity are 
+important to include if you plan to use fewer categories than the US standard. 
+
+Another option is to attempt a crosswalking approach if one or more of your 
+input datasets use different categories than the standard. This would allow 
+you to retain all of the data inputs, while keeping some granularity in categories. 
+
+**Multiracial Groups**
+
+Part of the US standard approach is allowing people to select as many race and 
+ethnicity categories as they identify with. Many people have multiracial 
+identities and capturing this is important. However, it creates a statistical 
+issue without an easy answer - how do you handle overlapping groups? 
+
+To understand the options below, let's provide an example of a single 
+person who selected both "Black or African American" and "Asian" for 
+their race. 
+
+The first option is to include all 
+relevant race/ethnicity groups. So for this example, you would have 
+groups for "Black or African American", "Asian", and "Both Black or African 
+American and Asian". If your dataset is large 
```

Comment:
```suggestion
combinations of race/ethnicity groups. So for this example, you would have 
groups for "Black or African American alone", "Asian alone", and "Both Black or African 
American and Asian". If your dataset is large 
```

Just an idea -- I think these "alone" terms are commonly used in government data.